hugo.toml

```diff
@@ -111,8 +111,6 @@ k3s_version = "v1.31.2+k3s1"
 kairos_version = "v3.2.3"
 # renovate: datasource=github-releases depName=kairos-io/osbuilder
 osbuilder_version = "v0.401.0"
```

Comment:
we could remove this too, it's not used anywhere, but it could be used to refer to osbuilder as the controller.